ParquetCSVImporter/Shims/BlockShim.cs

```diff
@@ -0,0 +1,52 @@
+﻿using ParquetClassLibrary.Sandbox.ID;
```

Comment:
comment: I'm nervous about needing to update this file any time Blocks are updated and any time Parquet Parents are updated, but I guess as long as Block doesn't have any optional constructor parameters this'll at least yell at you during compile time. I feel like there's something off about the Shim approach, having seen this, but I don't have constructive insight as to what to replace it with.

Reply:
I super super agree.
This is part of why I hesitated so long before adopting this approach.  But, for now it is the best I can do.